### DISCUSSION TITLE
Is it possible to add exceptions like *.png ?

### DISCUSSION THREAD

**Discussion by IlyasAlmyashev** [darkreader/darkreader#7694] [Q&A]
Hello. I have a weird  bug - when I open images in a browser, it leaves a "ghost" image on my display, even after I close the tab and the browser itself. It disappears over some time, but it is still annoying. I can't screenshot it, I can only see it on my monitor.

I tried  to add exceptions for pictures like *.png , but it seems that these patterns do not work.

**Comment by Gusted** [comment]
I'm sorry, but I'm confused by what you're asking and what you're trying to solve.

So, your monitor has ghost images(is this caused by dark reader?).
So why are you trying to add exceptions to pictures? (They don't solve ghosting on it's own).

Regards,
Gusted
